services/core-web/src/index.js

```diff
@@ -1,45 +1,103 @@
 import "core-js/stable";
 import "regenerator-runtime/runtime";
 
-import React, { Component } from "react";
+import React, { useState } from "react";
+import { ReactKeycloakProvider } from "@react-keycloak/web";
+import { useIdleTimer } from "react-idle-timer";
 import { render } from "react-dom";
 import { Provider } from "react-redux";
+import keycloak, { keycloakInitConfig } from "./keycloak";
+import Loading from "@/components/common/Loading";
 
-import { ENVIRONMENT } from "@mds/common";
-import { MatomoProvider, createInstance } from "@datapunt/matomo-tracker-react";
 import App, { store } from "./App";
 import "antd/dist/antd.less";
 import "./styles/index.scss";
 import fetchEnv from "./fetchEnv";
+import { unAuthenticateUser } from "./actionCreators/authenticationActionCreator";
 
-let instance = {};
-
-class Index extends Component {
-  constructor() {
-    super();
-    this.state = { environment: false };
-    fetchEnv().then(() => {
-      instance = createInstance({
-        urlBase: ENVIRONMENT.matomoUrl,
-        enableLinkTracking: false,
-        siteId: 1,
-      });
-      this.setState({ environment: true });
-    });
-  }
-
-  render() {
-    if (this.state.environment) {
-      return (
-        <Provider store={store}>
-          <MatomoProvider value={instance}>
-            <App />
-          </MatomoProvider>
-        </Provider>
-      );
+const idleTimeout = 60_000;
+const refreshTokenBufferSeconds = 60;
+
+export const Index = () => {
+  const [environment, setEnvironment] = useState(false);
+
+  fetchEnv().then(() => {
+    setEnvironment(true);
+  });
+
+  const { isIdle } = useIdleTimer({
+    timeout: idleTimeout,
+    throttle: 500,
+    crossTab: true,
+    syncTimers: 1000,
+    events: [
+      "mousemove",
+      "keydown",
+      "wheel",
+      "DOMMouseScroll",
+      "mousewheel",
+      "mousedown",
+      "touchstart",
+      "touchmove",
+      "MSPointerDown",
+      "MSPointerMove",
+      "visibilitychange",
+      "focus",
+    ],
+  });
+
+  const handleAuthErrors = (err = "") => {
+    console.log("Authentication error", err);
```

Comment:
I'd prefer to see authentication errors in the console, would be interested in other approaches!